src/layouts/AppLayout.vue

```diff
@@ -24,6 +24,28 @@ export default {
   components: {
     AppHeader,
     AppNavigation
+  },
+  watch: {
+    $route: function() {
+      // $nextTick = DOM updated
+      this.$nextTick(function() {
+        // set timeout needs a slight delay before referencing the DOM
+        setTimeout(() => {
```

Comment:
settimeout 0 looks a little strange to me. Did you run into issues without it? Doesn't the $nextTick callback take care of DOM not being fully rendered?